optaplanner-examples/src/main/resources/org/optaplanner/examples/conferencescheduling/optional/score/conferenceSchedulingConstraints.drl

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package org.optaplanner.examples.conferencescheduling.solver;
+package org.optaplanner.examples.conferencescheduling.score;
```

Comment:
```suggestion
package org.optaplanner.examples.conferencescheduling.optional.score;
```

Reply:
This is intentional so that constraint configuration continues to work. Constraint configuration enforces a package, and it is the package that I give.

Reply:
Could you please add a comment about that? It's not obvious and violates the consistency introduced in this PR.